MachObfuscator/DataAccess/Data+Structs.swift

```diff
@@ -17,6 +17,10 @@ extension Data {
         }
     }
 
+    func getStructs<T>(fromRange range: Range<Int>) -> [T] {
+        return getStructs(atOffset: range.startIndex, count: range.count / MemoryLayout<T>.stride)
```

Comment:
ok, lets leave it for another PR. `stride` is used also in `UnsafeRawPointer.readStructs`. Probably we should check if and how metadata in files are aligned, but for the ones used currently this was not an issue, as their size was so that they do not require additional padding to be aligned.